translations/en.i18n.json

```diff
@@ -8,7 +8,9 @@
       "fullscreen": "Fullscreen",
       "fullscreenExit": "Exit fullscreen",
       "rewind": "Seek backwards",
+      "secondsRewind": "Seek {{seconds}} seconds backwards",
```

Comment:
do we have a ticket for adding translations in other languages ?

Reply:
yes, created FEC-13468